meta-lmp-bsp/dynamic-layers/tegra/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bbappend

```diff
@@ -0,0 +1 @@
+do_install[depends] += "tegra-uefi-capsules:do_install"
```

Comment:
This will fail for other platforms, we need something like:
```
TEGRA_INSTALL_DEPENDS ?= ""
TEGRA_INSTALL_DEPENDS:tegra = "tegra-uefi-capsules:do_install"
do_install[depends] += "${TEGRA_INSTALL_DEPENDS}"
```

Reply:
thanks @quaresmajose , fixed!